ksonnet-util/kausal.libsonnet

```diff
@@ -252,7 +252,7 @@ k {
 
     // rbac creates a service account, role and role binding with the given
     // name and rules.
-    rbac(name, rules)::
+    rbac(name, rules, annotations={}, pullSecrets={})::
```

Comment:
I'd put the arguments in the same order as how they are referenced below.

Reply:
Thanks for the feedback!  Updated per your suggestion.